scripts/delete-pv.sh

```diff
@@ -16,11 +16,32 @@
 set -ex
 PV_NUM=${PV_NUM:-12}
 
-NODE_NAMES=$(oc get node -o name -l node-role.kubernetes.io/worker)
-if [ -z "$NODE_NAMES" ]; then
-    echo "Unable to determine node name with 'oc' command."
-    exit 1
+# OCP 4.18+
```

Comment:
Simply wondering if we want to deduplicate this code in a common place/function that is called in both scripts, but not a blocker.

Reply:
It's probably something that can be improved, yeah.